src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_blocked_opt.cpp

```diff
@@ -165,49 +186,66 @@ JitConstants EltwiseKernel_blocked_opt::MakeLoadJitConstants(const eltwise_param
             else
                 IE_ASSERT("MakeLoadJit : Unexpected dimension for eltwise optimized kernel.");
 
+            // Generate Jit
             switch (input.mode) {
                 case EltwiseInputMode::SCALAR:
                     jit.AddConstant(MakeJitConstant(name, input.scalar));
                     break;
                 case EltwiseInputMode::INPUT_BUFFER:
                 {
                     const std::string idx_order = "INPUT" + toCodeString(input.index) + "_IDX_ORDER";
+                    const std::string temp_vec_var = "\\\n\tMAKE_VECTOR_TYPE(ACCUMULATOR_TYPE, " + toCodeString(vec_size) + ")" +
+                                                     " tmp_a" + toCodeString(op_num) + "_" + toCodeString(input_idx) + " = ";
                     jit.AddConstant(MakeJitConstant(idx_order, default_indexing_str));
 
                     if (params.inputs[input.index].LogicalSize() == 1) {
                         const std::string vload_name = "DO_VLOAD" + toCodeString(op_num) + "_" + toCodeString(input_idx);
-                        const std::string vload_value = "\\\n\tMAKE_VECTOR_TYPE(ACCUMULATOR_TYPE, " + toCodeString(vec_size) + ") " +
-                                                        "tmp_a" + toCodeString(op_num) + "_" + toCodeString(input_idx) + " = " +
+                        const std::string vload_value = temp_vec_var +
                                                         "(MAKE_VECTOR_TYPE(ACCUMULATOR_TYPE, " + toCodeString(vec_size) + "))" +
                                                         "(input" + toCodeString(input.index) + "[0])";
                         jit.AddConstant(MakeJitConstant(vload_name, vload_value));
                         jit.AddConstant(MakeJitConstant(name, "tmp_a" + toCodeString(op_num) + "_" + toCodeString(input_idx)));
+                    } else if (feature_broadcasting) {
+                        // Load as scalar and broadcast to vector variable
```

Comment:
As I remember, it's not easy to debug the issue if the issue happen related this after a while, or hard to figure out it for jit code without source dump. So how about some examples comment for result of the strings?

Reply:
I added a case for broadcasting. It contained broadcasting+normal loading with global id. We might not need for each case.